tests/server/keyspace/keyspace_test.go

```diff
@@ -52,16 +58,20 @@ func TestKeyspaceTestSuite(t *testing.T) {
 func (suite *keyspaceTestSuite) SetupTest() {
 	ctx, cancel := context.WithCancel(context.Background())
 	suite.cancel = cancel
-	cluster, err := tests.NewTestCluster(ctx, 3, func(conf *config.Config, serverName string) {
+	cluster, err := tests.NewTestAPICluster(ctx, 3, func(conf *config.Config, serverName string) {
```

Comment:
Please move the mcs related test under mcs package.

Reply:
I think it is more related to keyspace.